### PR TITLE
Add TDP 40 and Fix Headers

### PR DESCRIPTION
**Pull request by mzero2nine**
Hi @dimsome no urgency here, but when you come back from your Easter Vacation, could you please review and merge? 

Thanks very much! :))

### CODE REVIEW

File: TDP/tdp-40.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+Given the track record, we believe it is most wholesome for the mStable project to once again deploy our own liquidity on Uniswap v3 and consolidate the current effort with Gamma Strategies one very strong and liquid pair, with easy entry and exit opportunities across the DeFi ecosystem.
+
+The manual process involved combined with the unavoidable realization of some impermanent loss from Ondo Finance made us understand that their offer is currently too risky, given the economic and geopolitical sentiment we have today.
+
+Instead, we will solve this issue once and for all with a bullish range deployment of MTA on Uniswap v3, which also will give the treasury a lot of upside exposure to ETH, and will eventually arrive at the position owning only ETH, which can then be further utilized in the space as we deploy an even more bullish fresh MTA position on Uniswap v3, and at which stage the Gamma position should hold a much healthier portion of MTA [than is currently the case](../assets/TDP-40/gamma-mta.png).
```
Comment (dimsome) [inline TDP/tdp-40.md:38]: this is now just a link to the image. I think we should include the image just here, right? `caption`

**Review by dimsome — CHANGES_REQUESTED** [review]
Added one comment :)

File: TDP/tdp-40.md (hunk 1 above)
Comment (mzero2nine) [reply TDP/tdp-40.md:38]: A link to the image was the original draft. Not sure it has to be a part of the TDP? It's not like it's essential to the TDP in any way? 🤔

Comment (dimsome) [reply TDP/tdp-40.md:38]: hmm ok

**Review by dimsome — APPROVED** [review]
ok lgtm

**Pull request merged**